ghedesigner/tests/expected_demo_results.json

```diff
@@ -1,54 +1,54 @@
 {
   "find_design_bi_rectangle_constrained_single_u_tube": {
-    "active_borehole_length": 132.79936866478178,
+    "active_borehole_length": 133.4910467664394,
     "number_of_boreholes": 74
   },
   "find_design_bi_rectangle_double_u_tube_series": {
-    "active_borehole_length": 127.28198187827503,
+    "active_borehole_length": 128.61593151552975,
     "number_of_boreholes": 77
   },
   "find_design_bi_rectangle_single_u_tube": {
-    "active_borehole_length": 121.79696691388936,
+    "active_borehole_length": 122.43431450783052,
     "number_of_boreholes": 88
   },
   "find_design_bi_zoned_rectangle_single_u_tube": {
-    "active_borehole_length": 134.0152903408635,
+    "active_borehole_length": 134.87726908080376,
     "number_of_boreholes": 68
   },
   "find_design_near_square_coaxial": {
-    "active_borehole_length": 134.89227618557592,
-    "number_of_boreholes": 132
+    "active_borehole_length": 122.76544339220567,
+    "number_of_boreholes": 144
   },
   "find_design_near_square_double_u_tube": {
-    "active_borehole_length": 130.86452342334576,
+    "active_borehole_length": 130.55095231298313,
     "number_of_boreholes": 144
```

Comment:
Test results are updated. As I noted previously, the results are very similar to what they were before, except for a few of the coaxial tests, as shown here. This is primarily due to the bug uncovered where the fluid resistance instead of the combined pipe-grout resistance value was being used in the radial model.

https://github.com/BETSRG/GHEDesigner/pull/112/files#diff-445bcad92a6dfded274e894bb6adead4f430237182a47459c3747453eee628f2L159